src/server/cli/options.ts

```diff
@@ -15,22 +15,29 @@
  */
 
 import { Option } from "commander";
-import { DEFAULT_PORT } from "../models/server-settings/server-settings";
+import { DEFAULT_LOGGER_FORMAT, DEFAULT_PORT } from "../models/server-settings/server-settings";
 import { parseInteger } from "./utils";
 
 export const portOption = new Option(
   "-p, --port <number>",
   `Port number to start server on. Default: ${DEFAULT_PORT}`
 ).argParser(parseInteger);
 
+export const loggerOption = new Option(
+  "--logger-format <format>",
+  `Format for logger. Default: ${DEFAULT_LOGGER_FORMAT}`
```

Comment:
Could we print available formats?

Reply:
Commander takes care of it for us!

You could be tempted to ask: why commander doesn't take care of default value. Because default is calculated deep in `ServerSettings`. CLI should pass undefined, so we can merge it with config file and then pick default value if necessary.